integr8ly/delorean
# Del 242 test containers

To verify:

1. Make sure a RHMI cluster is available and running, and you have access to the `kubeconfig` file and it is saved locally somewhere.
2. Check out the branch and build it. Run the following command:
    ```
    export KUBECONFIG=<path to kubeconfig file>
     ./delorean tests run --output /tmp/tests --test-config ./test-containers.yaml
    ```
3. Wait for it to complete. It will take about 15m for the tests containers to complete. When they are finished, you should be able to access the results and logs in the output directory.

Review this commit: 0e1169029be09aa9398c36834febd871700443b1
